frontend/components/auction/debt/DebtAuctionBidTransactionTable.vue

```diff
@@ -0,0 +1,161 @@
+            <div>Auction fixed bid</div>
+            <div>
+                <FormatCurrency v-if="auction.bidAmountDai" :value="auction.bidAmountDai" currency="DAI" />
+                <span v-else class="opacity-50">Unknown</span>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Current compensation</div>
+            <div>
+                <FormatCurrency v-if="auction.receiveAmountMKR" :value="auction.receiveAmountMKR" currency="MKR" />
+                <span v-else class="opacity-50">Unknown</span>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Next lower compensation</div>
+            <button
+                v-if="auction.nextMaximumLotReceived"
+                class="ClickableText"
+                :disabled="!isActive"
+                @click="setInputBidAmount(undefined)"
+            >
+                <FormatCurrency :value="auction.nextMaximumLotReceived" currency="MKR" />
+            </button>
+            <span v-else class="opacity-50">Unknown</span>
+        </div>
+        <div class="flex justify-between items-center">
+            <div>Desired compensation</div>
+            <div class="flex w-1/2 items-center space-x-2 justify-end -mr-1">
+                <div class="w-full flex-shrink-0">
+                    <bid-input
+                        :input-bid-amount.sync="inputBidAmount"
+                        :min-value="auction.nextMaximumLotReceived"
+                        :fallback-value="auction.nextMaximumLotReceived"
+                        currency="MKR"
+                        :disabled="!isActive"
+                        :validator="validator"
+                    />
+                </div>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Combined Transaction Fees</div>
+            <div>
+                <FormatCurrency v-if="auction.combinedBidFeesDai" :value="auction.combinedBidFeesDai" currency="DAI" />
+                <span v-else class="opacity-50">Unknown</span>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Price on Uniswap</div>
+            <div>
+                <format-currency
+                    v-if="auction.marketUnitPrice && isActive"
+                    :value="auction.marketUnitPrice"
+                    :decimal-places="6"
+                    currency="DAI"
+                />
+                <span v-else class="opacity-50">Unknown</span> per MKR
+            </div>
+        </div>
+        <div class="flex justify-between font-bold">
+            <div>Price after the bid</div>
+            <div>
+                <format-currency
+                    v-if="unitPriceAfterBid && isActive && !isBidAmountNaN"
+                    :value="unitPriceAfterBid"
+                    :decimal-places="6"
+                    currency="DAI"
+                />
```

Comment:
Don't we need to show price in MKR here @LukSteib?

Reply:
Since the underlying exchange is providing DAI for MKR, IMO it's correct to show the price as `DAI per MKR`. So reverse than what we do for the surplus auctions

Reply:
> Since the underlying exchange is providing DAI for MKR

I think we changed it to fetch proper value of MKR per DAI, since we're effectively selling DAI here, right?